tests/test_views.py

```diff
@@ -4583,5 +4529,269 @@ def test_confirm_user_signal_called_when_user_confirms_email(self):
         assert_equal(mock_signals.signals_sent(), set([auth.signals.user_confirmed]))
 
 
+class TestDraftRegistrationViews(OsfTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(TestDraftRegistrationViews, self).setUp(*args, **kwargs)
+
+        self.user = AuthUserFactory()
+        self.auth = self.user.auth
```

Comment:
Since we're using an AuthUserFactory, consider getting rid of this instance variable and using self.user.auth directly.

The advantage would be improving readability of any tests that looked at privilege differences for two different users- it would be more immediately obvious which user was associated with `auth`.